frontend/translations/en.po

```diff
@@ -3904,7 +3904,7 @@ msgstr "Copy properties"
 
 #: src/app/main/ui/workspace/sidebar/shortcuts.cljs:95
 msgid "shortcuts.create-component"
-msgstr "Create component"
```

Comment:
is better to create a complete new translation string that reuse the old one, because this can be already translated to some langs, and the change could not reach those languages.